### PR TITLE
AnyBar widget

### PR DESCRIPTION
**Pull request by narma**
Hi!

It's a port of https://github.com/tonsky/AnyBar.

### CODE REVIEW

**Comment by narma** [comment]
Oh, it requires python 3.4 for asyncio. It's ok or we need to support 3.3 too?

**Comment by enkore** [comment]
I'm not exactly sure how to handle this. I think it would be ok to say "this needs either Python 3.4+, or 3.3+ with asyncio installed" (asyncio => https://pypi.python.org/pypi/asyncio). In this case, to fix the CI/build issue, see http://docs.enkore.de/i3pystatus/module.html#handling-dependencies (i.e. "asyncio" goes into MOCK_MODULES)

File: i3pystatus/anybar.py [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+            self.__class__.loop_thread.start()
+
+    def run(self):
+        self.output = {
+            "full_text": "●",
+            "color": self.colors.get(self.color, self.color)
+        }
+
+
+def anybar_remove(status, anybar):
```
Comment (enkore) [inline i3pystatus/anybar.py]: What is this for?

**Comment by narma** [comment]
I've updated PR with using regular socket/threading approach.

**Pull request merged**